### ISSUE TITLE
Here's a JSON file with population and FIPS data

### ISSUE THREAD

**Issue by Zoooook** [nytimes/covid-19-data#180]
tldr: https://github.com/Zoooook/CoronavirusTimelapse/blob/master/static/population.json

In the [page source](view-source:https://www.nytimes.com/interactive/2020/us/coronavirus-us-cases.html) for the [NYTimes U.S.A. map](https://www.nytimes.com/interactive/2020/us/coronavirus-us-cases.html) there is a chunk of json with fips and population data, however it is incomplete, listing only counties with confirmed cases. These population numbers appear to be using [this database](https://data.democratandchronicle.com/american-community-survey) as a source, so I filled it in with the rest of the counties.

There are a few anomalies. As you probably know, the 5 boroughs have been combined into a single entity, New York City, and the NYTimes assigns this a FIPS value of 36998. Also, Kansas City, MO has been created with a FIPS value of 29998, borrowing population from Clay, Jackson, and Platte counties in MO (but not Cass county, in fact). The county populations all still add up to the correct total state populations. The 50 states plus D.C. add up to the total U.S. population from that database.

The NYTimes json assigns FIPS values of xx999 to Unknown counties. For U.S. states, these have no population and so they are not included in this file. However, Puerto Rico (Unknown county) is included with a FIPS value of 72999, representing the total population of PR. The U.S. Virgin Islands, Guam, and the Northern Mariana Islands (Unknown counties, FIPS values 78999, 66999, 69999) were not in the same database, nor were their populations in the NYTimes json, as they are not represented on the map. I filled in 2018 population numbers for these from https://www.worldometers.info/.

There are also a few discrepancies between the NYTimes numbers and the source dataset that are not explained by combinations of counties; the NYTimes simply undercounts the population for a single county in 7 states. In these cases, I've corrected the 'population' attribute but included the incorrect number in the 'nyt_population' attribute.

The 'region' and 'subregion' attributes should always match state and county names from this repo. There is one non-ASCII character in Doña Ana County, New Mexico.

**Comment by jharrison** [comment]
@Zoooook I'm using your population file and I found two inconsistencies in the way things are named. I manually fixed them but thought you might want to know so you can do the same.

1) Consistent with what I've seen in other sources, municipalities do not include "Municipality" in the name. Except for Skagway Municipality. I changed it to Skagway and now it matches the rest.

2) Independent cities are listed like "Fairfax city" and this is consistent. Counties are listed without the county, like "Fairfax" for Fairfax County. This is also consistent. The one snag I found was that there's a county in Virginia called James City County. To be consistent, it should be listed as "James City" but its listed as "James city" instead. 

Summary:

- Change "Skagway Municipality" to "Skagway"
- Change "James city" to "James City"

**Comment by Zoooook** [comment]
Thanks for the heads up, those are both good catches

**Comment by tiffehr** [comment]
Closing this! It is listed in our Additive Data collection on our Project board, along with other Issues where people wanted FIPS or other geographic descriptive information or encodings.  We'll point people to that collection as this question arises again, including to your Issue.  Thank you!

**Comment by wesmor1231** [comment]
These numbers no longer seem to be the one that they are using on their page, does anyone have the new population files?

**Comment by Zoooook** [comment]
And they've added Joplin, MO and a bunch of counties in Puerto Rico and maybe some other places, so my file is definitely out of date.
